CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change Log
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 0.3
```

Comment:
Revert these changes.

I change version only in release commit.

Reply:
Ok, understood, pardon!
now i will check how works if set  zero queries i think there have bug with new return,
i'll fix it and make new commit again, thank you🤝